src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1880,6 +1880,17 @@ public static bool IsLabelContext(this SyntaxTree syntaxTree, int position, Canc
                 }
             }
 
+            // var(|
+            // var(id, |
+            if (token.IsKind(SyntaxKind.OpenParenToken, SyntaxKind.CommaToken))
+            {
+                if (token.Parent.Parent.IsKind(SyntaxKind.InvocationExpression) &&
```

Comment:
exact same code as above.  This should be extracted out into a helper "IsInvocationOfVarExpression" or something like that.